AmethystAPI/include/minecraft/src-client/common/client/renderer/screen/ScreenContext.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include "common/client/game/ClientInstance.h"
+#include "common/client/gui/gui/GuiData.h"
```

Comment:
Weird in here

Reply:
I dont understand

Reply:
You could be refactor them

Reply:
I suppose? I just left them in as they were